### PR TITLE
Wen elements variable names changed according to team conventions

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

File: src/main/java/trello/keys/NamePages.java [hunk 1]
```diff
@@ -42,7 +41,6 @@ public NamePages(final User user) {
      */
     public String getHomePage() {
```
Comment (raulchoque-eng) [inline src/main/java/trello/keys/NamePages.java]: Change the name of this method please. E.G.  getUriHomePage()

File: src/main/java/trello/ui/pages/BoardPage.java [hunk 2]
```diff
@@ -26,16 +26,16 @@
 public class BoardPage extends BasePage {
 
     @FindBy(css = ".placeholder")
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/BoardPage.java]: Replace css by className for find WebElement please.

File: src/main/java/trello/ui/pages/BoardPage.java [hunk 3]
```diff
@@ -26,16 +26,16 @@
...
-    private WebElement newListBtn;
+    private WebElement newListButton;
 
     @FindBy(name = "name")
-    private WebElement newListNameTxtBox;
+    private WebElement newListNameField;
 
     @FindBy(css = ".primary")
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/BoardPage.java]: Replace css by className for find WebElement please.

File: src/main/java/trello/ui/pages/BoardPage.java [hunk 4]
```diff
@@ -26,16 +26,16 @@
...
-    private WebElement addNewListBtn;
+    private WebElement addNewListButton;
 
     @FindBy(css = "a[class='js-close-list']")
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/BoardPage.java]: Replace css by className for find WebElement please.

File: src/main/java/trello/ui/pages/LoginPage.java [hunk 5]
```diff
@@ -27,42 +27,44 @@
...
 
     @FindBy(id = "password")
-    private WebElement passwordTxt;
+    private WebElement passwordField;
 
     @FindBy(css = SIGN_OFF_ID)
-    private WebElement passwordHiddenTxt;
+    private WebElement passwordHiddenField;
 
     private static final String SIGN_OFF_ID = "div.show-when-password.hidden";
```
Comment (raulchoque-eng) [inline src/main/java/trello/ui/pages/LoginPage.java]: Replace css by className for find WebElement please.

File: src/main/java/trello/keys/NamePages.java (hunk 1 above)
Comment (MelissaRoman06) [reply src/main/java/trello/keys/NamePages.java]: done.

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 2 above)
Comment (MelissaRoman06) [reply src/main/java/trello/ui/pages/BoardPage.java]: done.

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/trello/ui/pages/BoardPage.java]: done.

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 3 above)
Comment (MelissaRoman06) [reply src/main/java/trello/ui/pages/BoardPage.java]: done.

File: src/main/java/trello/ui/pages/BoardPage.java (hunk 4 above)
Comment (MelissaRoman06) [reply src/main/java/trello/ui/pages/BoardPage.java]: done.

**Review by JosueRodriguezGarcia — APPROVED** [review]

**Pull request merged**